lib/MTT/Values/Functions/MPI/OMPI.pm

```diff
@@ -165,10 +165,10 @@ sub find_network {
 sub get_version {
     my $bindir = shift;
 
-    open INFO, "$bindir/ompi_info --parsable|";
+    open INFO, "$bindir/ompi_info -V|";
```

Comment:
maybe get_version should try old way 1st and if it fails - try 2nd way.
To keep backwards compat